### PR TITLE
feat: #846 user information card new design style

### PR DESCRIPTION
**Pull request by J0taFerreira**
- user Identity card are now using the new design

some styling is on the common repo.

### CODE REVIEW

File: frontend/src/components/grantaccess/UserIdentityCard.vue [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+import { toRaw } from 'vue';
 import type { IdimProxyIdirInfo } from 'fam-api';
 import Card from 'primevue/card';
+import Icon from '../common/Icon.vue';
+import { IconSize } from '@/enum/IconEnum';
 
 const props = defineProps<{
     userIdentity: IdimProxyIdirInfo;
 }>(); // Vue3 alternative way for Type the defineProps().
+console.log(toRaw(props.userIdentity))
```
Comment (MCatherine1994) [inline frontend/src/components/grantaccess/UserIdentityCard.vue:11]: let'e remove the console log

**Review by ianliuwk1019 — CHANGES_REQUESTED** [review]
The PR title seems to be wrong at the ticket number (feat: #841).

**Review by ianliuwk1019 — CHANGES_REQUESTED** [review]
Just quickly look at it, will check later.
But where are the first name and last name?
Or is the scope being reduced?

File: frontend/src/components/grantaccess/UserIdentityCard.vue (hunk 1 above)
Comment (J0taFerreira) [reply frontend/src/components/grantaccess/UserIdentityCard.vue:11]: sorry, I thought I had removed the console.logs.

**Comment by J0taFerreira** [comment]
> The PR title seems to be wrong at the ticket number (feat: #841).

it was, thanks for noticing

File: frontend/src/components/grantaccess/UserIdentityCard.vue [hunk 2]
```diff
@@ -1,24 +1,32 @@
 <script setup lang="ts">
+import { toRaw } from 'vue';
 import type { IdimProxyIdirInfo } from 'fam-api';
 import Card from 'primevue/card';
+import Icon from '../common/Icon.vue';
```
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/components/grantaccess/UserIdentityCard.vue:5]: Sure, thanks!

**Comment by MCatherine1994** [comment]
> Just quickly look at it, will check later.
> But where are the first name and last name?
> Or is the scope being reduced? My initially understanding is this might depend on backend change.

Hi Ian, we have another ticket https://app.zenhub.com/workspaces/fsa-fingerprint-61f04f08304d3e001a4f2578/issues/gh/bcgov/nr-fam-idim-lookup-proxy/40 to let IDIM proxy return these information, maybe we work on the frontend and backend detail later once we have the data?

**Comment by ianliuwk1019** [comment]
> > Just quickly look at it, will check later.
> > But where are the first name and last name?
> > Or is the scope being reduced? My initially understanding is this might depend on backend change.
> 
> Hi Ian, we have another ticket https://app.zenhub.com/workspaces/fsa-fingerprint-61f04f08304d3e001a4f2578/issues/gh/bcgov/nr-fam-idim-lookup-proxy/40 to let IDIM proxy return these information, maybe we work on the frontend and backend detail later once we have the data?

Thank you very much Catherine!! Sounds good, I see and understand the new ticket.
* Then I think (nr-fam-idim-lookup-proxy/40 is not difficult) and for this ticket frontend can still have "**First Name**" and "**Last Name**" in the table columns as ticket (Figma) intended but for missing values (from new ticket nr-fam-idim-lookup-proxy/40) we can use "**N/A**" for now (it will only be **temporary** for few days in main branch). So this ticket can still be done but will need to **be released together** with ticket nr-fam-idim-lookup-proxy/40.
* In ticket **nr-fam-idim-lookup-proxy/40** we can consider include **fam backend** (not the proxy repo) change to return correct results set for frontend. This also includes update **client-code-gen** for Swagger (OpenAPI).
image

**Comment by ianliuwk1019** [comment]
Maybe next time, but the branch name is missing "/":
image

**Comment by ianliuwk1019** [comment]
Interesting.... I am seeing this:
image
Is it because style is on common repo, that's why I see distorted card?

**Comment by MCatherine1994** [comment]
Tested with the latest shared stylesheet in the common theme repo, looks good. Chatted with Ian, create a follow up ticket #921 to implement first name, last name frontend and backend code. This one should be good to merge.

**Comment by ianliuwk1019** [comment]
> Tested with the latest shared stylesheet in the common theme repo, looks good. Chatted with Ian, create a follow up ticket #921 to implement first name, last name frontend and backend code. This one should be good to merge.

Thank you Catherine! Feel free to approve it and merge.

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**